transbank/webpay/webpayplus/exceptions/transaction_exception.py

```diff
@@ -0,0 +1,47 @@
+
+class TransactionException(Exception):
+    pass
+
+
+class TransactionHttpException(TransactionException):
```

Comment:
The other SDKs handle exceptions on each action, for example `Transaction.create` would rise a `CreateTransactionException` in Java, a `TransactionCreateException` in PHP or `TransactionCreateException` in .net

Reply:
In the case that the REST API responds with an HTTP error code you should rise an exception in the method the user calls from the transaction classes and pass that HTTP error code through. 

For example, if an user calls `Transaction.create` and something fails, the risen exception should be `TransactionCreateException` and should reflect the HTTP error code or else a -1